helm/templates/promtail/deployment.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: extensions/v1beta1
```

Comment:
could you add support for resources request / limit?

Reply:
Sure I will add it tomorrow

Reply:
Done, I've added it for loki and promtail